src/js/components/Popup/Popup.stories.tsx

```diff
@@ -31,7 +31,7 @@ export const PopupActions = () => (
       <PopupAction
         text={<Icon name="more-horiz" />}
         actions={actions}
-        direction={['down', 'center']}
+        direction={['down', 'overlay']}
```

Comment:
Does the change from `center` to `overlay` introduce some regression issues? I think it would be a good idea to just make sure that anywhere that uses `PopupAction` doesn't use  `{['down', 'center']}. If so be sure to have a PR ready to make those changes before pushing v4

Reply:
I changed from `center` to `overlay` cause there is no `center` option... Maybe someone remove it and forgot to update the stories